### PR TITLE
Compose updated to beta01-1.0.0

### PR DESCRIPTION
**Pull request by parshav**

### CODE REVIEW

File: screendataui/src/main/java/com/pv/screendataui/views/SDLabel.kt [hunk 1]
```diff
@@ -83,6 +83,7 @@ fun SDLabel(label: SomeLabel) {
                 text = it,
                 fontSize = 12.sp,
                 color = textColor
+//                fontFamily = fontFamily(Typeface.DEFAULT_BOLD)
```
Comment (0xLeif) [inline screendataui/src/main/java/com/pv/screendataui/views/SDLabel.kt:86]: 👀

File: screendataui/src/main/java/com/pv/screendataui/views/SDImage.kt [hunk 2]
```diff
@@ -44,6 +46,7 @@ fun SDImage(image: SomeImage) {
 
     Image(
         imageSource.value,
+        "",
```
Comment (0xLeif) [inline screendataui/src/main/java/com/pv/screendataui/views/SDImage.kt:49]: Nice

File: build.gradle [hunk 3]
```diff
@@ -1,15 +1,15 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext {
-        compose_version = '1.0.0-alpha09'
+        compose_version = '1.0.0-beta01'
```
Comment (0xLeif) [inline build.gradle:4]: 😎

**Review by 0xLeif — APPROVED** [review]

File: screendataui/src/main/java/com/pv/screendataui/views/SDLabel.kt (hunk 1 above)
Comment (parshav) [reply screendataui/src/main/java/com/pv/screendataui/views/SDLabel.kt:86]: Im not even sure tbh

**Pull request merged**